cluster_generator/codes.py

```diff
@@ -86,49 +100,50 @@ def setup_gamer_ics(ics, regenerate_particles=False, use_tracers=False):
         gamer_ptypes.insert(0, "tracer")
     gamer_ptype_num = {"tracer": 0, "dm": 2, "star": 3}
     hses = [ClusterModel.from_h5_file(hf) for hf in ics.profiles]
-    parts = ics._generate_particles(
-        regenerate_particles=regenerate_particles)
-    outlines = [
-        f"Merger_Coll_NumHalos\t\t{ics.num_halos}\t# number of halos"
-    ]
+    parts = ics._generate_particles(regenerate_particles=regenerate_particles)
+    outlines = [f"Merger_Coll_NumHalos\t\t{ics.num_halos}\t# number of halos"]
     for i in range(ics.num_halos):
         particle_file = f"{ics.basename}_gamerp_{i+1}.h5"
         if ics.num_particles["star"][i] == 0:
             ptypes = gamer_ptypes[:-1]
         else:
             ptypes = gamer_ptypes
-        write_amr_particles(parts[i], particle_file, ptypes, gamer_ptype_num,
-                            in_cgs=True, format="hdf5")
+        write_amr_particles(
+            parts[i], particle_file, ptypes, gamer_ptype_num, in_cgs=True, format="hdf5"
+        )
         hse_file_gamer = ics.profiles[i].replace(".h5", "_gamer.h5")
-        hses[i].write_model_to_h5(hse_file_gamer, overwrite=True,
-                                  in_cgs=True, r_max=ics.r_max)
+        hses[i].write_model_to_h5(
+            hse_file_gamer, overwrite=True, in_cgs=True, r_max=ics.r_max
+        )
         vel = ics.velocity[i].to_value("km/s")
         outlines += [
             f"Merger_File_Prof{i+1}\t\t{hse_file_gamer}\t# profile table of cluster {i+1}",
             f"Merger_File_Par{i+1}\t\t{particle_file}\t# particle file of cluster {i+1}",
             f"Merger_Coll_PosX{i+1}\t\t{ics.center[i][0].v}\t# X-center of cluster {i+1} in kpc",
             f"Merger_Coll_PosY{i+1}\t\t{ics.center[i][1].v}\t# Y-center of cluster {i+1} in kpc",
             f"Merger_Coll_VelX{i+1}\t\t{vel[0]}\t# X-velocity of cluster {i+1} in km/s",
-            f"Merger_Coll_VelY{i+1}\t\t{vel[1]}\t# Y-velocity of cluster {i+1} in km/s"
+            f"Merger_Coll_VelY{i+1}\t\t{vel[1]}\t# Y-velocity of cluster {i+1} in km/s",
         ]
     mylog.info("Write the following lines to Input__TestProblem: ")
     for line in outlines:
         print(line)
-    num_particles = sum([ics.tot_np[key] for key in ics.tot_np])
+    # num_particles = sum([ics.tot_np[key] for key in ics.tot_np])
```

Comment:
I noticed on my own that this line isn't used, so you can go ahead and delete it.